components/camel-opentelemetry/src/test/resources/log4j2.properties

```diff
@@ -24,6 +24,6 @@ appender.out.name=out
 appender.out.layout.type=PatternLayout
 appender.out.layout.pattern=%d [%-15.15t] %-5p %-30.30c{1} - %m%n
 logger.opentelemetry.name=org.apache.camel.opentelemetry
-logger.opentelemetry.level=INFO
+logger.opentelemetry.level=DEBUG
```

Comment:
I think should be INFO again

Reply:
sure, let's just see if tests pass in CI